processor/src/test/java/org/mapstruct/ap/test/generics/typebounds/WildcardedInterface.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.generics.typebounds;
+
+/**
+ * @author Ben Zegveld
+ */
+public interface WildcardedInterface {
```

Comment:
I understand that this is the base for a wildcard.. But the interface itself does not contain wildcards. Hence the name is somewhat confusing. Would `UpperBound` be a [better name](http://www.angelikalanger.com/GenericsFAQ/FAQSections/TypeArguments.html#What%20is%20a%20bounded%20wildcard?)?

Reply:
Or `WildCardBase` or just `WildCard`

Reply:
went with `WildCard` and `WildCardImpl`